src/main/kotlin/racingcar/ui/RaceGameController.kt

```diff
@@ -0,0 +1,36 @@
+package racingcar.ui
+
+import racingcar.domain.Car
+import racingcar.domain.CarNameParser
+import racingcar.domain.Race
+import racingcar.domain.RaceResult
+import racingcar.domain.exception.IncorrectGameSettingException
+import racingcar.ui.view.InputView
+import racingcar.ui.view.ResultView
+
+class RaceGameController {
+    fun main() {
+        val nameOfCarInput = InputView.getNameOfCarInput()
+        val totalCountOfRoundInput = InputView.getTotalCountOfRoundInput()
```

Comment:
변수명에 `input`이라는 의미를 꼭 포함킬시킬 필요가 있을까요? 🤔  
입력 받은 값인지 아닌지 알 필요가 없다고 생각해요. 😃 

```suggestion
        val names = InputView.getNameOfCarInput()
        val totalCountOfRound = InputView.getTotalCountOfRoundInput()
```